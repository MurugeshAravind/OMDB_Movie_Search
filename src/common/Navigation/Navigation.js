import React from "react"
import styled from "styled-components"

const Nav = styled.nav`
    font-size: 1.5em;
    text-align: center;
    color: black;
`

const Navigation = () => {
return <Nav>
  <h1>OMDB Movie Search</h1>
</Nav>
}
export default Navigation