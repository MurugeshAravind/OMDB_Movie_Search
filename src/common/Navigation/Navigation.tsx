import styled from 'styled-components';

const Nav = styled.nav`
  font-size: 1.5em;
  text-align: center;
  color: black;
`;

const Header = styled.h1`
  color: silver;
`;

const Navigation = () => {
  return (
    <Nav>
      <Header>OMDB Search</Header>
    </Nav>
  );
};
export default Navigation;
