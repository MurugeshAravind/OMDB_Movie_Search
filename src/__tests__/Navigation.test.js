import React from "react";
import { render } from "@testing-library/react";
import { createMemoryHistory } from "history";
import Router from "react-router-dom";
import Navigation from "../common/Navigation/Navigation";
import renderer from "react-test-renderer";
import "jest-styled-components";

const history = createMemoryHistory();
const navElement = renderer.create(<Navigation />)
const NavigationComponentInstance = () => {
    <Router history={history.location} navigator={history}>
     <Navigation history={history} />
    </Router>
}
describe('navigation test', () => {
    test('test nav', () => {
       render(NavigationComponentInstance)
       expect(NavigationComponentInstance).toBeTruthy();
    })
    test('nav snapshot test', () => {
        expect(navElement).toMatchSnapshot();
    })
})