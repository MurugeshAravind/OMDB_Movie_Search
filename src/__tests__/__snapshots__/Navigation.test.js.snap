// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navigation test nav snapshot test 1`] = `
<nav
  className="fRzOdK"
>
  <h1>
    OMDB Movie Search
  </h1>
</nav>
`;
